CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -53,6 +53,15 @@
  */
 #elif defined ( __ICCARM__ )
 
+  /* CMSIS compiler control architecture macros */
```

Comment:
It's not part of IAR 7.8 that we use, neither earlier versions. I can edit this to check if its not already defined, then it would be valid for any IAR version.